### PR TITLE
Upgrade Fibers to npm:latest for all platforms [1.0.8]=>[1.0.10], whi…

### PR DESCRIPTION
**Pull request by lukeadams**
Meteor currently uses a fork of Node-Fibers for Windows, and a (minorly) out of date version for everything else. The Windows fork was created to fix thread searching [https://github.com/meteor/node-fibers/commit/d519f0c5971c33d99c902dad346b817e84bab001]. The author has integrated the fix into master [https://github.com/laverdet/node-fibers/commit/91c4b4020303379c4c50346882f39a201279f592], so the commit applied to meteor core [https://github.com/meteor/meteor/commit/466559a8c41ff8b623f56d1a04037a0c5ad485a7] is no longer needed.

TLDR: This commit will allow for simpler adoption of [future node versions](https://github.com/meteor/meteor/issues/5124#issuecomment-210968751) by upgrading one of the core Meteor deps, and using the same version for all platforms.

### CODE REVIEW

**Comment by lukeadams** [comment]
Welp, just saw this commit: https://github.com/meteor/meteor/commit/48b3169d55a0062e38f3db4a142e7ef64e4dcc29
Should have submitted a pull request earlier 😄 . I had all of these changes pushed to my repo already!

**Comment by stubailo** [comment]
Dang, you got scooped! Thanks for the PR but sounds like I should close this.

**Comment by lukeadams** [comment]
@stubailo Haha yeah I was working on upgrading node weeks ago, but never submitted a pull request (so stuff like this was done in parallel).
Oh well lesson learned

**Comment by benjamn** [comment]
Thanks anyway @lukeadams! By the way, I mentioned your helpful comments about the Node upgrade in my Meteor Night talk last week. I think the `meteor-promise` problems are fixed now, too.
